### ISSUE TITLE
cant repair evac floors

### ISSUE THREAD

**Issue by deltanedas** [space-wizards/space-station-14#15388]
## Description
since i guess its protected or something not able to place floors on lattice making it impossible to even try and repair minibomb
making plating should be excluded from protection since lattice cant be made so no sitting outside shittle

**Reproduction**
1. bomb evac
2. try to place tile over lattice
3. wont let you because protected

**Comment by deltanedas** [comment]
yes thats fine but you shouldnt block placing plating since that can be vital for fixing bombings
blocking lattices is perfectly fine

**Comment by crazybrain23** [comment]
If you can't fix bombs, then just block bombs too, nobody should be spacing the evac shuttle.

**Comment by VerinSenpai** [comment]
as far as I remember, this doesn't even stop people from doing that, since I'm pretty sure you can still place lattice so long as it isn't by another grid.

**Comment by K-Dynamic** [comment]
Should be resolved by #36989
